test-common/src/chain_env.rs

```diff
@@ -321,6 +321,19 @@ impl ChainEnv {
             app_state_bytes: serde_json::to_vec(&self.init_config).unwrap(),
             chain_id: TEST_CHAIN_ID.to_owned(),
             validators,
+            consensus_params: Some(ConsensusParams {
+                evidence: Some(EvidenceParams {
+                    max_age_duration: Some(::protobuf::well_known_types::Duration {
+                        seconds: 172_800,
```

Comment:
some explanation of this number
how about const variable?

Reply:
It's just 48 hours in seconds, the default config value of tendermint.
Only used in unit tests.